frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -328,6 +329,13 @@ const SampleDetailsContent = ({
         <TabPane tab={`Lineage`} key="6" style={tabStyle}>
           <SampleDetailsLineage sample={sample} />
         </TabPane>
+
+        // Only display the Pool tab if the sample is a pool
+        {sample.is_pool &&
```

Comment:
So far we've adopted the position to always show all tabs and mention they are not required in them inside the tab if the tab does not apply. Maybe our position could be revised but until then to keep things uniform i would move this test and add it inside the Pane to display a message in case this is not a pool.

Reply:
Done